.tools/test/stacks/plugin/typescript/plugin_stack.ts

```diff
@@ -42,6 +43,7 @@ class PluginStack extends cdk.Stack {
       this.initSubscribeSns(sqsQueue, snsTopic);
       this.batchMemory = acctConfig[`${toolName}`]?.memory ?? "16384";
       this.batchVcpus = acctConfig[`${toolName}`]?.vcpus ?? "4";
+      this.batchStorage = acctConfig[`${toolName}`]?.storage ?? "20";
```

Comment:
It'd be nice if there was a comment or something identifying the units.

Reply:
Done